### ISSUE TITLE
Idea: allow disabling anonymous comments in the moderator API/settings file

### ISSUE THREAD

**Issue by vdboor** [django-fluent/django-fluent-comments#6]
When sites have a nice login/registration system, it's useful to disable anonymous comments. This should be configurable in a settings variable.

**Comment by jimlyndon** [comment]
Old issue. Was this accomplished in any fashion?

**Comment by vdboor** [comment]
No, this is still open as an idea to implement - both on the backend (via the moderator?) and the frontend (nice UI to redirect to the login).

**Comment by julianfox** [comment]
still open? if yes +1, i am working on it for a project :-)

**Comment by kdahlhaus** [comment]
It was easy to do by checking user now that the form is a different tag.  I can show what I mean when I am back at the office.
